CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED]
+
+### Bug Fixes
+
+* **aws-apigateway:** fix validation of resources failing for proxy resource paths (`{proxy+}`)
```

Comment:
We haven’t updated our CONTIBUTING guide yet, but this is not needed anymore. Instead just make sure your squashed commit to master adheres to conventionalcommits. Sorry for the trouble.